### PR TITLE
docs: add a security policy

### PR DESCRIPTION
**Pull request by eseidel**
This came up as part of a customer review.

### CODE REVIEW

**Review by eseidel — COMMENTED** [review]
I'm sure there is much more we can add, just putting down the basics for now.

File: security.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Security
+
+This is our public security policy, which is a subset of our security practices.
+
+This document exists both for education of our employees as well as for
+reference by our customers.  Changes are welcome and should be reviewed by Eric.
+
+Shorebird is an software application.  Most of our code is open source and
```
Comment (bryanoltman) [inline security.md:8]:
```suggestion
Shorebird is a software application.  Most of our code is open source and
```

File: security.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+receive security training as part of their onboarding process, including reading
+this document.
+
+## Access Control
+
+### Passwords
+
+All passwords are stored in a password manager.  We use
+[1Password](https://1password.com/).  We do not store passwords in code, in
+emails, or in any other insecure location.
```
Comment (bryanoltman) [inline security.md:34]:
```suggestion
emails, or in any other insecure location. Passwords should also not be entered in plaintext in any CLI, as they will then be stored in the shell history file.
```

File: security.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+infrastructure.
+
+Shorebird only offers a hosted service at this time.  We do not offer [on-prem
+or cloud-prem at this
+time](https://github.com/shorebirdtech/shorebird/issues/485).
+
+## Terms
+
+Throughout this document we will refer to the following terms:
+- Customer / you: A user of Shorebird.
```
Comment (erickzanardo) [inline security.md:19]: Should we refrain from `you` and keep to `customer`? No strong argument other than to use a single term per "thing"

**Review by bryanoltman — APPROVED** [review]

File: security.md (hunk 3 above)
Comment (eseidel) [reply security.md:43]: I'm going to keep it for now, thanks!

**Comment by eseidel** [comment]
PTAL.

**Pull request merged**